### ISSUE TITLE
Explore data that can be visualised on Leeds Music Education Partnership

### ISSUE THREAD

**Issue by taz-chiles** [open-innovations/leeds-schools-cultural-engagement#38]
This is blocked on receiving additional data from Leeds City Council. 

- [ ] Taz to follow up with LCC this week 

Once we received this, we can explore the data and see what can be visualised on the site. 

- [ ] Identify key data to visualise on a new 'Leeds Music Education Partnership' page
- [ ] Create a mockup page to share and receive feedback from Briony 
- [ ] Finalise the page and visualisations 

This is low priority given how close we are to the deadline for final wrap up on the 21st August, so we won't worry about it if we don't receive the data in enough time to create the visualisations. However it has been mentioned in initial discussions so I will raise it with Briony.

**Comment by taz-chiles** [comment]
Closing as not planned
